plugins/lang/python/python_loader.cc

```diff
@@ -497,6 +501,127 @@ runPythonScript(std::shared_ptr<PythonContext> titl,
     };
 }
 
+void
+PythonPlugin::
+injectMldbWrapper(PythonSubinterpreter & pyControl)
+{
+    std::string code = R"code(
+
+
+class mldb_wrapper(object):
+
+    @staticmethod
+    def wrap(mldb):
+        return Wrapped(mldb)
+
+
+    class Exception(Exception):
+        pass
+
+    class ResponseException(Exception):
+        def __init__(self, response):
+            self.response = response
+
+        def __str__(self):
+            return ('Response status code: {r.status_code}. '
+                    'Response text: {r.text}'.format(r=self.response))
+
+    class Response(object):
+
+        import json as jsonlib
+
+        def __init__(self, url, raw_response):
+            self.url         = url
+            self.headers     = {k: v for k, v in
+                                raw_response.get('headers', {})}
+            self.status_code = raw_response['statusCode']
+            self.text        = raw_response.get('response', '')
+            self.raw         = raw_response
+
+            self.apparent_encoding = 'unimplemented'
+            self.close             = 'unimplemented'
+            self.conection         = 'unimplemented'
+            self.elapsed           = 'unimplemented'
+            self.encoding          = 'unimplemented'
+            self.history           = 'unimplemented'
+            self.iter_content      = 'unimplemented'
+            self.iter_lines        = 'unimplemented'
+            self.links             = 'unimplemented'
+            self.ok                = 'unimplemented'
+            self.raise_for_status  = 'unimplemented'
+            self.reason            = 'unimplemented'
+            self.request           = 'unimplemented'
```

Comment:
@nicolaskruchten I left it there to show the list that requests Response object has. We can remove them if needed.